src/decorator/CircuitBreakerDecorator.ts

```diff
@@ -0,0 +1,39 @@
+import { CircuitBreaker } from '../CircuitBreaker';
+import { CircuitBreakerState } from '../CircuitBreakerState';
+
+type CBConfig = {
```

Comment:
In order to keep consistency in the library namings, could you rename this type to `CircuitBreakerConfiguration` ?